recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
```

Comment:
I also find this strange, the compilers should not be necessary since this is basically a repack.

Reply:
@boldorider4 @skupr-anaconda We should avoid changing anything unless absolutely required. Here, compilers can inject things into the runtime requirements, etc.

Reply:
I asked because I noticed in logs:
`WARNING (cuda-nvdisasm): dso library package defaults/linux-64::libstdcxx-ng==11.2.0=h1234567_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)`
Souldn't the target package be responsible for runtime requirements? Sorry, for the question, I'm far of understanding yet how this CUDA stuff works

Reply:
I've asked this upstream: https://github.com/conda-forge/cuda-nvdisasm-feedstock/issues/9
However it's a common pattern in these feedstocks and no doubt there's a reason for it. Nevertheless, they're happy to receive improvements on them.
They're written by people at nvidia and we're just copying them with minimal adaptations so that we can rebase our changes in future releases as easily as possible

Reply:
So it's their way of including some of the run requirements of the repackaged binaries: https://github.com/conda-forge/cuda-nvdisasm-feedstock/issues/9
IMO it's not causing enough of a problem to remove them (it'd have to be done in a range of feedstocks), what do you think?

Reply:
We should leave this there. Like I said, we should diverge from their recipes only if absolutely required.

Reply:
we could submit PRs to the conda-forge recipes, though